src/main/java/org/apache/commons/validator/routines/checkdigit/Modulus97CheckDigit.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.validator.routines.checkdigit;
+
+import java.io.Serializable;
+
+import org.apache.commons.validator.GenericTypeValidator;
+import org.apache.commons.validator.GenericValidator;
+
+/**
+ * MOD 97-10 module according to ISO/IEC 7064, MOD 97-10.
+ * <p>
+ * See <a href="https://de.wikipedia.org/wiki/ISO/IEC_7064">Wikipedia - ISO/IEC_7064 (de)</a>
+ * for more details.
+ * </p>
+ *
+ * <p>
+ * This MOD 97-10 module can be used to validate the LEI (Legal Entity Identifier), ICD id 0199
+ * and in VATidBECheckDigit (VATIN_BE), and also in LeitwegValidator
+ *
+ * @since 1.10.0
+ */
+public class Modulus97CheckDigit extends AbstractCheckDigit implements Serializable {
```

Comment:
Don't implement Serializable, we are moving away from that IMO, folks that need serialization of some kind can use serialization proxies; see "Effective Java" by J. Block.

Reply:
Cannot remove it.
Error:  Failures: 
Error:    Modulus97CheckDigitTest>AbstractCheckDigitTest.testSerialization:300 org.apache.commons.validator.routines.checkdigit.Modulus97CheckDigit error during serialization: java.io.NotSerializableException: org.apache.commons.validator.routines.checkdigit.Modulus97CheckDigit

Reply:
Hello @homebeaver 
That was a bug in our test ;-) Please rebase on master to pick up https://github.com/apache/commons-validator/commit/fdc8ee9c26f668192b34ec5b955b3adf47f98b2d

Reply:
after rebase and rempve implement Serializable:
```
[INFO] --- spotbugs-maven-plugin:4.8.6.4:check (default-cli) @ commons-validator ---
[INFO] BugInstance size is 1
[INFO] Error size is 0
[INFO] Total bugs: 1
Error:  Medium: Class org.apache.commons.validator.routines.CodeValidator defines non-transient non-serializable instance field checkdigit [org.apache.commons.validator.routines.CodeValidator] In CodeValidator.java SE_BAD_FIELD
[
```

Reply:
Ah, I see: Because `CodeValidator implements Serializable` and `CodeValidator` has a `CheckDigit` field, we are stuck with `CheckDigit` implementing `Serializable` if we want to honor the `Serializable` contract on `CodeValidator`. I guess we'll have to drop serialization in a major version bump if we want to go that way.

TY for cycling through this one @homebeaver ! 😄

Reply:
I revert the commit 8add96b5939ef1cc2d3ab3ff8d774ee48f194d73